static/js/compose_ui.js

```diff
@@ -413,3 +418,47 @@ export function get_compose_click_target(e) {
     }
     return e.target;
 }
+
+export function on_compose_select_stream_update(new_value) {
+    const $stream_header_colorblock = $(".stream_selection_dropdown").find(
+        ".stream_header_colorblock",
+    );
+    stream_bar.decorate(new_value, $stream_header_colorblock, true);
+    compose.update_on_recipient_change();
+    // TODO(compose-dropdown)
```

Comment:
https://chat.zulip.org/#narrow/stream/6-frontend/topic/focus.20on.20composebox

Reply:
Looks like this got resolved in the linked thread.